logos/src/logos/main.py

```
@@ -3673,6 +3675,37 @@ async def update_model(data: GiveFeedbackRequest):
+            name=data.name,
+            description=data.description,
+            tags=data.tags,
+            parallel=data.parallel,
+            weight_privacy=data.weight_privacy,
+            weight_latency=data.weight_latency,
+            weight_accuracy=data.weight_accuracy,
+            weight_cost=data.weight_cost,
+            weight_quality=data.weight_quality,
+        )
+    if status == 200 and data.name:
+        asyncio.create_task(fetch_price_for_single_model(data.model_id, data.name))
```

Ruff (0.15.13)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)